### PR TITLE
ike hates fun

### PR DESCRIPTION
**Pull request by qwertyquerty**
## About The Pull Request

ike hates fun

## Why It's Good For The Game

ike hates fun

## Changelog
:cl:
del: ike hates shitcode, coderskirt removed
/:cl:

### CODE REVIEW

**Comment by qwertyquerty** [comment]
ike hates fun

File: code/modules/clothing/under/skirt_dress.dm [hunk 1]
```diff
@@ -99,19 +99,3 @@
 	icon_state = "plaid_green"
 	item_state = "plaid_green"
 	item_color = "plaid_green"
-
-/obj/item/clothing/under/dress/skirt/coder
```
Comment (qwertyquerty) [inline code/modules/clothing/under/skirt_dress.dm:103]: ike hates fun

**Comment by ro5490** [comment]
RIP coderskirt

**Comment by Archanial** [comment]
Ike hates fun

**Comment by park66665** [comment]
RIP codeskirt, you will be remembered...

**Pull request merged**